### PR TITLE
Add Umami icon

### LINKED ISSUE #7443: Umami
### Brand Name

Umami

### Website

https://umami.is

### Popularity Metric

The Similarweb rank is `1,000,102`. See https://www.similarweb.com/de/website/umami.is
But the [repo](https://github.com/mikecao/umami) has about `11.3k` stars.

### Official Resources for Icon and Color

- svg: https://github.com/mikecao/umami/blob/master/assets/logo.svg
- color: `#00000`, from the logo

### Additional Comments

- The [Repo](https://github.com/mikecao/umami) is licensed under the [MIT license](https://github.com/mikecao/umami/blob/master/LICENSE), so I think the logo is also licensed under this license.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by zrfisaac**
umami

**Issue:** Close #7443

**Similarweb rank:** [1,007,752](https://www.similarweb.com/website/umami.is/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Umami is a simple, easy to use, self-hosted web analytics solution. The goal is to provide you with a friendlier, privacy-focused alternative to Google Analytics and a free, open-sourced alternative to paid solutions.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11223,6 +11223,11 @@
             "hex": "18A5D6",
             "source": "https://ulule.frontify.com/d/EX3dK8qsXgqh/branding-guidelines"
         },
+        {
+            "title": "Umami",
+            "hex": "000000",
+            "source": "https://github.com/mikecao/umami/blob/master/assets/logo.svg"
```
Comment (sisp) [inline _data/simple-icons.json:11229]: Please use a permalink to the logo file as described in the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md):

> In any case, the commit hash should be part of the URL. On GitHub, you can get the correct URL by pressing `y` on the GitHub page you want to link to. You can get help at the [getting permanent links to files page](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files).

```suggestion
            "source": "https://github.com/mikecao/umami/blob/5358e28a68e0ccf4fdba95dbdc495779429f2a4f/assets/logo.svg"
```

**Review by sisp — CHANGES_REQUESTED** [review]
Thanks for your contribution, @zrfisaac! :pray:

I see a notable difference between the original and your icon:


Could you please check the icon again?

File: _data/simple-icons.json (hunk 1 above)
Comment (zrfisaac) [reply _data/simple-icons.json:11229]: Sorry! Thanks, a will take a another better look in the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md).

**Comment by zrfisaac** [comment]
strange, I will check again of couse, thanks for the observation! 👍

**Comment by MagicLike** [comment]
Are there any updates on this?

**Comment by zrfisaac** [comment]
> Are there any updates on this?

sorry, I will take a look, probably tomorrow.

**Comment by MagicLike** [comment]
No problem, thanks for taking the time! 😃

**Comment by zrfisaac** [comment]
@sisp I've made the changes, I gues that now i'ts okey.
Sorry for the delay, I will go back contrubute. :raised_hands:

**Review by sisp — CHANGES_REQUESTED** [review]
Thanks for the update, @zrfisaac!

The shapes of the original icon are not properly united now. There are path points of the ring within the fill area of the other shape. Please don't _join_/_combine_ the SVG paths but rather _unite_ them, so those obsolete path points are eliminated.
unami-points

**Comment by service-paradis** [comment]
@zrfisaac Do you plan to continue working on this one?

**Comment by zrfisaac** [comment]
Yep, I had some problems in my work that made me be a little off, but I'm going back to work in this project now, I hope this week I will post the fix.

**Comment by adamrusted** [comment]
@simple-icons/icon-review - have just updated the path, and added the MIT license. Could you please review?

**Comment by uncenter** [comment]
Are there any blockers for this anymore? Happy to help get this through.

**Review by service-paradis — APPROVED** [review]
LGTM 🎉

**Pull request merged**